src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/readme.md

```diff
@@ -10,15 +10,16 @@ Verifies that a provided software entitlement token grants permission to use a s
 | ------ | ------------------------------------------------------ |
 | POST   | {endpoint}/softwareEntitlements/?api-version={version} |
 
-Sample: `https://samples.westus.batch.azure.com/softwareEntitlements/?api-version=2017-01-01.3.1`
+Sample: `https://samples.westus.batch.azure.com/softwareEntitlements/?api-version=2017-05-01.5.0`
 
-| Placeholder | Type   | Description                                                                     |
-| ----------- | ------ | ------------------------------------------------------------------------------- |
-| endpoint    | string | The Batch account url endpoint supplied by Azure Batch via environment variable |
-| version     | string | The API version of the request                                                  |
+| Placeholder | Type   | Description                                                                      |
+| ----------- | ------ | -------------------------------------------------------------------------------- |
+| endpoint    | string | The Batch account url endpoint supplied by Azure Batch via environment variable. |
+| version     | string | The API version of the request. <p/> Must be `2017-05-01.5.0`.                   |
```

Comment:
Will require maintaining to add new api versions as they come out -  might be better to say "must be '2017-01-01.5.1' or a more recent version, API versions are listed @ https://docs.microsoft.com/en-us/rest/api/batchservice/batch-service-rest-api-versioning